import os
os.system('git pull')
import PyInit_error32_enc
